### DISCUSSION TITLE
[세미나 제안] 해외 기업 코딩인터뷰에 관심 있으신분 계실까요?

### DISCUSSION THREAD

**Discussion by wooyaggo86** [ThinkAboutSoftware/AcademicConference#116] [General]
안녕하세요.지난주 GDG와 F-lab이라는 곳을 통해서 코딩인터뷰 관련된 주제로 몇 가지 팁을 이야기할 기회가 있었는데요.
학술회의에서도 관심 있으신분들이 계시면, 여기에서도 나눔하고 싶어 글 올립니다.
괜찮으시다면 다음주(독서모임이 없는 주) 금요일(2월 4일)로 생각하고 있어요.
발표시간은 1시간 정도가 될 것 같고, 발표자료는 아래 링크에서 보실 수 있습니다.
관심있으시면 댓글 부탁드립니다~

https://github.com/wooyaggo86/Presentation/blob/main/20210123/(%EB%82%98%EB%A7%8C%20%EB%AA%B0%EB%9E%90%EB%8D%98)%20FFANG%20%EC%BD%94%EB%94%A9%20%EC%9D%B8%ED%84%B0%EB%B7%B0%EC%9D%98%20%EB%B9%84%EB%B0%80.md?plain=1

**Comment by TaeHyoungKwon** [comment]
저 관심있습니다~!! 😄  좋은 경험 공유 감사드립니다 🙇

**Comment by hi-hyein** [comment]
저도 관심 있습니다~!
좋은 발표 공유 너무 감사합니다 💘

**Comment by momoci99** [comment]
감사합니다!!!!!!!!!!

**Comment by jongfeel** [comment]
급하게 달다 보니 태형님 글에 댓글로 달아버렸군요 ㅋㅋㅋㅋ

--- 

와우 이런 좋은 발표도 해 주시고 감사합니다.

저도 상당히 관심 많습니다.

**Comment by jongfeel** [comment]
벌써 네 분이나 모였으니 빠르게 캘린더에 일정 만들어 보겠습니다

2월 4일 금요일 오후 11시 게더타운!

**Reply by wooyaggo86** [reply]
감사합니다^^👍

**Reply by wooyaggo86** [reply]
GDG모임에서 제안해주셔서 할 용기가 났습니다ㅎㅎ

**Comment by jongfeel** [comment]
동훈님은 여기에 따로 코멘트를 주셔서...
동훈님 포함하면 다섯명입니다!

https://github.com/ThinkAboutSoftware/AcademicConference/issues/123#issuecomment-1024155094
